* add node_io support to default:furnace the is re-implemented in pipeworks



diff --git a/compat-furnaces.lua b/compat-furnaces.lua
index 492332a..97cc79c 100644
--- a/compat-furnaces.lua
+++ b/compat-furnaces.lua
@@ -431,3 +431,7 @@ minetest.register_node(":default:furnace_active", {
 	after_dig_node = pipeworks.after_dig
 })
 
+if minetest.get_modpath("node_io") then
+	node_io.init_default_furnace("default:furnace")
+	node_io.init_default_furnace("default:furnace_active")
+end
diff --git a/depends.txt b/depends.txt
index 0cb7406..572fd01 100644
--- a/depends.txt
+++ b/depends.txt
@@ -2,3 +2,4 @@ default
 mesecons
 mesecons_mvps
 digilines?
+node_io?
